### PR TITLE
[tracy] enable fibers feature

### PR DESCRIPTION
**Pull request by jhruby**
Adding fibers feature support for tracy library.

If this PR updates an existing port, please uncomment and fill out this checklist:

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [N/A] SHA512s are updated for each updated download.
- [N/A] The "supports" clause reflects platforms that may be fixed by this new version.
- [N/A] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [N/A] Any patches that are no longer applied are deleted from the port's directory.
- [N/A] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [N/A] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/tracy/vcpkg.json
Comment (WangWeiLin-MV) [inline ports/tracy/vcpkg.json:1]:
```diff
-  "port-version": 1,
+  "port-version": 2,
```
Please update the `port-version`.
Running `./vcpkg format-manifest --all` to format.
Running `./vcpkg x-add-version --all` to update the version database.

**Review by WangWeiLin-MV — CHANGES_REQUESTED** [review]
Note: I will be converting your PR to draft status. The above suggested changes are only recommendations. If you are willing to adopt them, please click "ready for review" after making the modifications. If you do not wish to make any changes, please click "ready for review" directly. That way, I can be aware that you've responded since you can't modify the tags.

File: versions/t-/tracy.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "0cabe3789ff10b958ebe1622adb09a5c19addca3",
```
Comment (WangWeiLin-MV) [inline versions/t-/tracy.json:4]: Re-running `./vcpkg x-add-version --all` to fix the version database.

**Review by WangWeiLin-MV — CHANGES_REQUESTED** [review]

**Comment by jhruby** [comment]
Is there any action from me required to merge this in?

**Review by BillyONeal — APPROVED** [review]
Fibers are often 'alternatives' rather than features but the manual for this makes it clear that it's just a feature:

https://github.com/wolfpld/tracy/blob/3d70a614368d8a60db557c8cd5b6a759fb55b7ba/manual/tracy.tex#L1664

**Review by BillyONeal — APPROVED** [review]

**Review by WangWeiLin-MV — APPROVED** [review]

**Comment by WangWeiLin-MV** [comment]
Port usage tests pass with following triplets:
* x64-windows

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**